manual/getting_started/types.md

```diff
@@ -0,0 +1,180 @@
+# Types
+
+The types of the on-chain world are declared in a given
+[Substrate](https://substrate.io/)-based chain's (Rust) source code. While many
+types may remain consistent across chains, many may differ. On one chain,
+`AccountData` may be defined with fields describing fungible assets; on another
+(hypothetical) chain, perhaps `AccountData` describes non-fungible assets,
+reputation, linked accounts or something else entirely. Capi's codegen outputs
+TypeScript-idiomatic equivalents of the Rust-declared types, along with
+utilities such as object and variant factories, type guards and runtime
+[scale-ts](https://github.com/paritytech/scale-ts) codecs. This reduces the
+friction of targeting a Rust-defined environment from your JavaScript
+environment. It also ensures optimal type safety!
+
+## Conversion Table
+
+The following table describes how Capi translates a given chain's Rust-declared
+types to and from JavaScript types.
+
+<!-- https://github.com/paritytech/capi/pull/73 -->
+
+<!-- TODO(@tjjfvi): which of these conversions is/are now invalid? -->
+
+| Rust                 | TypeScript                   |
```

Comment:
@tjjfvi after refactoring, I remembered a convo from a while ago about leaving each column as a standalone code fence (instead of cells for each type). The issue with this is vertical alignment. Meanwhile, doing it all with HTML is so unattractive. Any thoughts on the best way to copy over [your table](https://github.com/paritytech/docs.capi.dev/blob/dc2d2729a0698e9b4a0582bd7edba86f826a3cac/docs/basics/types.md?plain=1#L214-L285)?

Reply:
Perhaps a picture? We could place it in `server/static`... or maybe we should just move that file to a new `static` root dir?